app/javascript/elm/src/Main.elm

```diff
@@ -172,6 +171,7 @@ init flags url key =
         , isSearchAsIMoveOn = flags.isSearchAsIMoveOn
         , overlay = Overlay.init flags.isIndoor
         , scrollPosition = flags.scrollPosition
+        , status = Active
```

Comment:
can you check what happens when you save and open a link with fixed dormant sessions? Cause I think it's not gonna work if you don't send it in the flags.

Reply:
you're right, thx